pkgs/applications/office/qownnotes/default.nix

```diff
@@ -19,14 +19,14 @@
 let
   pname = "qownnotes";
   appname = "QOwnNotes";
-  version = "24.1.2";
+  version = "24.1.4";
 in
 stdenv.mkDerivation {
   inherit pname appname version;
```

Comment:
```suggestion
  inherit pname version;
```

Reply:
How is the need for `appname` satisfied, then?

Reply:
It's used later in the derivation.

Reply:
Yes, that's what I meant. 😁

Reply:
Not necessary to inherit for `appname` to be available.

Reply:
Ah, nice. Thank you!